docs/how-to/hip_runtime_api/memory_management/host_memory.rst

```diff
@@ -0,0 +1,205 @@
+.. meta::
+  :description: This chapter describes the host memory of the HIP ecosystem
+                ROCm software.
+  :keywords: AMD, ROCm, HIP, host memory
+
+.. _host_memory:
+
+********************************************************************************
+Host memory
+********************************************************************************
+
+Host memory where on the host (e.g. CPU) of the machine in random access memory
+(RAM). The host memory has three different main types in HIP:
+
+* Pageable memory
+* Pinned memory
+* Unified Memory
```

Comment:
Saying the host memory has three different "types" makes it sound as if it was different hardware. Also pageable and pinned/non-pageable memory are not exclusive to HIP or CUDA, that's a general concept.
```suggestion
Host memory is "normal" memory as allocated by C or C++, and resides in the RAM of the host.
Host memory can be allocated in two different ways:

* Pageable memory
* Pinned memory
```

Also unified memory is not something that's only related to host memory, it should get its own section